stubs/networkx/networkx/classes/multidigraph.pyi

```diff
@@ -15,23 +16,24 @@ from networkx.classes.reportviews import (
 
 __all__ = ["MultiDiGraph"]
 
+# NOTE: Graph subclasses relationships are so complex
+# we're only overriding methods that differ in signature from the base classes
+# to use inheritance to our advantage and reduce complexity
 class MultiDiGraph(MultiGraph[_Node], DiGraph[_Node]):
     @cached_property
     def succ(self) -> MultiAdjacencyView[_Node, _Node, dict[str, Any]]: ...
     @cached_property
     def pred(self) -> MultiAdjacencyView[_Node, _Node, dict[str, Any]]: ...
     @cached_property
-    def edges(self) -> OutMultiEdgeView[_Node]: ...  # type: ignore[override]
-    # Returns: OutMultiEdgeView
+    def edges(self) -> OutMultiEdgeView[_Node]: ...
     @cached_property
     def out_edges(self) -> OutMultiEdgeView[_Node]: ...
     @cached_property
-    def in_edges(self) -> InMultiEdgeView[_Node] | InMultiEdgeDataView[_Node, _EdgeWithData[_Node]]: ...  # type: ignore[override]
-    # Returns : InMultiEdgeView or InMultiEdgeDataView
+    def in_edges(self) -> InMultiEdgeView[_Node] | InMultiEdgeDataView[_Node, _EdgeWithData[_Node]]: ...
```

Comment:
I think this property is always `InMultiEdgeView` ([source](https://github.com/networkx/networkx/blob/c5f46466d7f27fd16e4f3b59a44c0b9cca443caf/networkx/classes/multidigraph.py#L710C16-L710C31)).

The [documentation](https://networkx.org/documentation/stable/reference/classes/generated/networkx.MultiDiGraph.in_edges.html#networkx.MultiDiGraph.in_edges) says "InMultiEdgeView or InMultiEdgeDataView" because it is documented as a method rather than a callable property. It is already annotated that the `__call__` of `InMultiEdgeView` may return `InMultiEdgeDataView`, which comports with the documentation.

This change, if accepted, should be propagated to the `DiGraph` class as well.